tests/e2e/test_copy_project.py

```diff
@@ -395,13 +395,13 @@ def test_copy_project_e2e(kili, src_project):
         for asset_src, asset_new in zip(assets_src, assets_new):
             assert asset_src["externalId"] == asset_new["externalId"]
 
-            if Path(asset_src["content"]).is_file():
```

Comment:
Comment ca peut écrire un fichier en local avec un nom d'URL ?

Reply:
L'erreur sur windows:
`FAILED tests/e2e/test_copy_project.py::test_copy_project_e2e - OSError: [WinError 123] The filename, directory name, or volume label syntax is incorrect: 'https:\\storage.googleapis.com\\label-public-staging\\asset-test-sample\\video\\short_video.mp4'`

Je pense que c'est un bug pathlib de python 3.7, parce que sous 3.11 ça passe sans souci:
```text
Python 3.11.0 (main, Oct 24 2022, 18:26:48) [MSC v.1933 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> from pathlib import Path
>>> Path('https:\\storage.googleapis.com\\label-public-staging\\asset-test-sample\\video\\short_video.mp4').is_file()
False
>>>
```

Reply:
Le bug est que pathlib sur windows en 3.7 il interprete les string 'https:\\storage.googleapis.com\\...\\short_video.mp4' comme existant en local alors que c'est pas le cas c'est ca ?
tu as vérifié que ca aurait pas d'impact dans le code des fonctions,  autre que dans ce test ?

Reply:
J'ai regardé dans ProjectCopier et MediaDownloader, c'est bon normalement

Reply:
https://bugs.python.org/issue42855